gaztec/src/config.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  createLogger,
+  createAztecNodeClient,
+  type PXE,
+  AztecNode,
+} from "@aztec/aztec.js";
+import { PXEService } from "@aztec/pxe/service";
+import { PXEServiceConfig, getPXEServiceConfig } from "@aztec/pxe/config";
+import { KVPxeDatabase } from "@aztec/pxe/database";
+import { KeyStore } from "@aztec/key-store";
+import { L2TipsStore } from "@aztec/kv-store/stores";
+import { createStore } from "@aztec/kv-store/indexeddb";
+import { BBWASMLazyPrivateKernelProver } from "@aztec/bb-prover/wasm/lazy";
+import { WASMSimulator } from "@aztec/simulator/client";
```

Comment:
This makes me very sad - we shouldn't need these many things to create pxe

Reply:
While imports could be tidier, PXE is still a beast. The fact that you can configure it fully it's good IMO!